zadanie_kalkulator.py

```diff
@@ -45,4 +49,7 @@ def divide(x, y):
     elif choice == '4':
```

Comment:
posłużmy się tutaj słownikiem żeby zamienić tego ifa na "emulowany" switch statement. https://stackoverflow.com/questions/60208/replacements-for-switch-statement-in-python
Chcemy jako wartości w kluczu mieć referencje do funkcji, jako klucze chcemy mieć liczby mówiące o działaniu. Wybieramy funkcję na podstawie inputu użytkownika i wykonujemy funkcję, i wypisujemy wynik funkcji